test/testOpenwhisk.js

```diff
@@ -140,4 +140,44 @@ describe('Testing OpenWhisk adapter', () => {
       },
     });
   });
+
+  it('openwhisk adds JSON request body.content to payload.content', async () => {
+    const params = {
+      __ow_headers: {
+        'content-type': 'application/json',
+      },
+      __ow_logger: log,
+      __ow_method: 'post',
+    };
+    let payload = {};
+
+    // adds JSON from request body.content to payload.content
+    params.body = '{ "content": { "body": "# Foo" } }';
```

Comment:
openwhisk already invokes the action with the json decoded.

Reply:
also, this should probably just be `params.body = '# Foo';`

as openwhisk promotes the json properties as params. for example:

```
$ curl https://${APIHOST}/api/v1/web/guest/demo/hello.json -H 'Content-Type: application/json' -d '{"name":"Jane"}'
{
  "response": {
    "name": "Jane",
    "__ow_method": "post",
    "__ow_headers": {
      "accept": "*/*",
      "connection": "close",
      "content-length": "15",
      "content-type": "application/json",
      "host": "172.17.0.1",
      "user-agent": "curl/7.43.0"
    },
    "__ow_path": ""
  }
}
```